.rubocop.yml

```diff
@@ -24,6 +24,9 @@ Metrics/LineLength:
   Exclude:
     - 'app/controllers/catalog_controller.rb'
 
+Metrics/AbcSize:
+  Exclude:
+    - 'app/helpers/content_block_helper.rb'
```

Comment:
It's the `editable_content_block` method in particular, which has a score of 16 (the threshold is 15). Extracting `form_for` into its own method, like you suggested, may do the trick. I'll have a go at this later.